src/psyclone/tests/domain/gocean/transformations/gocean1p0_transformations_test.py

```diff
@@ -1025,13 +1025,6 @@ def test_omp_do_schedule_guided(tmpdir):
     assert GOcean1p0Build(tmpdir).code_compiles(psy)
 
```

Comment:
Not your fault but pylint reports that the test functions at lines 605 and 557 have duplicate names. Please could you take a look?

Reply:
True, they seem to do the same so I removed one.